spark/src/main/resources/reference.conf

```diff
@@ -1 +0,0 @@
-geotrellis.accumulo.catalog = "metadata"
```

Comment:
We actually need this. This is a default that will be pulled into the user's `application.conf`, even if they don't supply one or unless they overwrite it.